savaslabs/savaslabs.github.io
# Issue #2817: Update teleconsole video

Hi @AnneTee -

This PR updates the teleconsole video for the post `/2017/03/20/lightning-q1-2017.html` to a more technical video that better showcases teleconsole. It uses an embedded iframe video from google drive.

## To test
- Pull this branch
- Go to `/2017/03/20/lightning-q1-2017.html` and check that the teleconsole video from Tip 1 is no longer the asciinema cast you and I did ( 😭 ) but is an updated video showing slack + terminal. 
- Check the video is centered and responsive!
- Lastly, please check that anyone on the internet can play the video by loading the page via an incognito browser (not signed into your Savas account). It resides in a drive folder that I've made public, but just wanted to check it isn't limited to the Savas Team only!

Thanks again for the help!
